template/info_label.txt

```diff
@@ -17,3 +17,5 @@
 13, spinal cord centerline, PAM50_centerline.nii.gz
 14, point-wise spinal level labels, PAM50_spinal_midpoint.nii.gz
 15, spinal cord mask with discrete spinal levels, PAM50_spinal_levels.nii.gz
+16, spinal dorsal nerve rootlets with discrete spinal levels, PAM50_rootlets.nii.gz
```

Comment:
I'm surprised! We were missing the PAM50_rootlets file in info_label.txt... but indeed, it's not in #19.